### PR TITLE
AV-204987 LBSvc status fix for dualstack IPs

### PR DESCRIPTION
**Pull request by aaha97**
this PR addresses the issue AKO was updating the same status with IPv4 IP and then IPv6 IP resulting in only one IP added to the status at a time.

### CODE REVIEW

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: internal/status/svc_status.go [hunk 1]
```diff
@@ -60,44 +60,43 @@ func (l *leader) UpdateL4LBStatus(options []UpdateOptions, bulk bool) {
...
 			}
+
 			for _, vip := range option.Vip {
-				service.Status = corev1.ServiceStatus{
-					LoadBalancer: corev1.LoadBalancerStatus{
-						Ingress: []corev1.LoadBalancerIngress{{
-							IP:       vip,
-							Hostname: svcHostname,
-						}}}}
+				service.Status.LoadBalancer.Ingress = append(
```
Comment (DixitAakash) [inline internal/status/svc_status.go:65]: @aaha97 This is tested to work for the first operation right? Just had this doubt, otherwise changes look good.

Comment (pkoshtavmware) [reply internal/status/svc_status.go:65]: Consider the case when we had an IpV4 and IPv6 already in place. Now if lets say request for updating IPV4 to a new Ip comes in. Will this still update the existing IPv4 or just append it at the end ?

**Comment by Dhivyaaj** [comment]
No JIRA Ids found for the PR. Jira id is mandatory to update fix version in jira.                                            Please update respective Jira id in PR title or commit message                                            if the PR is intented for default branches of repo.                                            For mandatory exemptions, comment trigger phrase 'skip jira-id-check' in PR.                                            For manual trigger, comment trigger phrase 'run jira-id-check' in PR.

File: internal/status/svc_status.go (hunk 1 above)
Comment (aaha97) [reply internal/status/svc_status.go:65]: @DixitAakash @pkoshtavmware have updated it to set list to empty first, thanks

Comment (pkoshtavmware) [reply internal/status/svc_status.go:65]: Is this tested ?
As in does rest layer send `[existingip, newip]` in `option.Vip` or just `[existingip]` .

**Comment by apalsule** [comment]
@aaha97 Please add a jira id to this.

**Review by pkoshtavmware — APPROVED** [review]

**Comment by pkoshtavmware** [comment]
build ako

**Review by DixitAakash — DISMISSED** [review]

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

File: tests/gatewayapitests/status/httproute_test.go [hunk 2]
```diff
@@ -681,8 +681,8 @@ func TestHTTPRouteUnprocessedGateway(t *testing.T) {
 	parentRefs := akogatewayapitests.GetParentReferencesV1([]string{gatewayName}, namespace, ports)
 	hostnames := []gatewayv1.Hostname{"foo-8080.com"}
 
-	akogatewayapitests.SetupHTTPRoute(t, httpRouteName, namespace, parentRefs, hostnames, nil)
 	akogatewayapitests.SetupGateway(t, gatewayName, namespace, gatewayClassName, nil, listeners)
+	akogatewayapitests.SetupHTTPRoute(t, httpRouteName, namespace, parentRefs, hostnames, nil)
```
[the remark this answers is not in the document]
Comment (aaha97) [reply tests/gatewayapitests/status/httproute_test.go]: this was discussed and the test case has been skipped for now

**Comment by aaha97** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Review by DixitAakash — APPROVED** [review]

**Pull request merged**